dataprofiler/profilers/float_column_profile.py

```diff
@@ -131,11 +141,8 @@ def _is_each_row_float(cls, df_series):
         :return: is_float_col
         :rtype: list
         """
-        len_df = len(df_series)
-        if len_df == 0:
-            return list()
-
-        return [NumericStatsMixin.is_float(x) for x in df_series]
+        if len(df_series) == 0: return list()
+        return df_series.map(NumericStatsMixin.is_float)
```

Comment:
do we consider the trade-off with memory here? List comprehension seems better in memory usage than map.

Reply:
Will be roughly the same regardless. Both require a second list to be created.